src/core/css/inc/mixins.scss

```diff
@@ -105,199 +97,6 @@
   }
 }
 
-/* Arrows */
-$default-arrow-margin: 3px;
-
-@mixin arrow($margin: $default-arrow-margin) {
-  background: url("~amo/img/icons/arrow-link-color.svg") 50% 50% no-repeat;
-  border-radius: $border-radius-m;
-  content: "";
-  display: inline-block;
-  height: 16px;
-  margin-right: -$margin * 2;
-  position: relative;
-  right: -$margin;
-  top: 4px;
-  width: 22px;
-
-  [dir=rtl] & {
-    left: -$margin;
-    margin-left: -$margin * 2;
-    margin-right: 0;
-    right: auto;
-    transform: rotate(180deg);
-  }
-}
-
-@mixin arrow-down($margin: $default-arrow-margin) {
-  @include arrow($margin);
-
-  transform: rotate(90deg);
-
-  [dir=rtl] & {
-    transform: rotate(270deg);
-  }
-}
-
-// This alias to arrow (which is the same as arrow-next) makes it easier to
-// differentiate two arrows when both arrow-next and arrow-previous are used
-// near each other.
-@mixin arrow-next($margin: $default-arrow-margin) {
-  @include arrow($margin);
-}
-
-@mixin arrow-previous($margin: $default-arrow-margin) {
-  @include arrow($margin);
-
-  left: -$margin;
-  margin-left: -$margin * 2;
-  margin-right: 0;
-  right: auto;
-  transform: rotate(180deg);
-
-  [dir=rtl] & {
-    left: auto;
-    margin-left: 0;
-    margin-right: -$margin * 2;
-    right: -$margin;
-    transform: none;
-  }
-}
-
-@mixin arrow-up($margin: $default-arrow-margin) {
-  @include arrow($margin);
-
-  transform: rotate(270deg);
-
-  [dir=rtl] & {
-    transform: rotate(90deg);
-  }
-}
-
-@mixin arrow-link-right-aligned() {
-  background-image: url("~amo/img/icons/arrow-link-color.svg");
-  margin-right: 0;
-  position: absolute;
-  right: 8px;
-  top: 14px;
-
-  [dir=rtl] & {
-    left: 8px;
-    right: auto;
-  }
-}
-
-/* Button */
-@mixin button(
-  $background,
-  $background-active: #fff,
-  $background-hover: #fff,
-  $border-color: transparent,
-  $border-radius: $border-radius-s,
-  $color: $white
-) {
-  @include font-regular();
-
-  background: $background;
-  border: 1px solid $border-color;
-  border-radius: $border-radius;
-  cursor: pointer;
-  display: inline-block;
-  margin: 0;
-  min-height: 39px;
-  padding: 10px 14px 8px;
-  text-align: center;
-  text-decoration: none;
-  text-shadow: 1px 1px 0 transparentize(invert($color), 0.5);
-  transition-duration: $transition-short;
-  transition-property: background-color border-color;
-  vertical-align: middle;
-
-  .Icon {
-    @include margin-end(6px);
-
-    height: 16px;
-    vertical-align: top;
-    width: 16px;
-  }
-
-  &.Button--small {
```

Comment:
This is still referenced in the Notice styles: https://github.com/mozilla/addons-frontend/blob/67da4111618ba06c0967e461929174597aeba7e6/src/ui/components/Notice/styles.scss#L44

Maybe it just needs to be renamed in `Notice/styles.scss`?